content/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/configuring-notifications-in-github-desktop.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Configuring notifications in GitHub Desktop
+shortTitle: Configuring notifications
+intro: '{% data variables.product.prodname_desktop %} will keep you up-to-date with notifications about events that occur in your pull request branch.'
+versions:
+  fpt: '*'
+---
+## About notifications in {% data variables.product.prodname_desktop %}
+
+{% data variables.product.prodname_desktop %} will show a system notification for events that occur in the currently selected repository. Notifications will be shown when:
+
+- Pull request checks have failed.
+- A pull request review is left with a comment, approval, or requested changes.
+
+Clicking the notification will switch application focus to {% data variables.product.prodname_desktop %} and provide more detailed information.
+
+## Pull request checks failed notifications
+
+When changes are made to a pull request branch you will receive a notification if the checks in your repository fail.
+
+![pull request checks failed notification](/assets/images/help/desktop/pull-request-checks-failed-notification.png)
+
+Clicking the notification will display a dialog with details about the checks. Once you've reviewed why the checks have failed you can re-run the checks, or quickly switch to the pull request branch to get started on fixing the errors. For more information, see "[Viewing and re-running checks in GitHub Desktop](/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/viewing-and-re-running-checks-in-github-desktop)."
```

Comment:
Feel free to ignore this very minor comma addition, I just like adding pauses in sentences 😄 
```suggestion
Clicking the notification will display a dialog with details about the checks. Once you've reviewed why the checks have failed, you can re-run the checks, or quickly switch to the pull request branch to get started on fixing the errors. For more information, see "[Viewing and re-running checks in GitHub Desktop](/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/viewing-and-re-running-checks-in-github-desktop)."
```